eth/main.cpp

```diff
@@ -425,6 +426,10 @@ int main(int argc, char** argv)
 		catch (...) {}
 	}
 
+	AccountManager accountm;
+	if (accountm.execute(argc, argv))
```

Comment:
I think it is saner to add the "fork" here, i.e. let main.cpp decide when to forward execution to AccountManager.